crates/sui-adapter-transactional-tests/tests/programmable/split_coins.exp

```diff
@@ -0,0 +1,60 @@
+processed 19 tasks
+
+init:
+A: object(100), B: object(101)
+
+task 1 'publish'. lines 9-29:
+created: object(106)
+written: object(105)
+
+task 2 'run'. lines 30-30:
+created: object(108)
+written: object(100), object(107)
+
+task 3 'view-object'. lines 32-34:
+Error: task 3, lines 32-32. Unbound fake id 109
```

Comment:
Looks like the view-object got off by 1 or two in these tests

Reply:
Yeah, removed one account from `init` and did not notice...